Documentation/demo.md

```diff
@@ -0,0 +1,231 @@
+### Demo
+
+#### Set up cluster
+
+<img src="https://storage.googleapis.com/etcd/gifs/01_etcd_clustering_2016050501.gif" alt="01_etcd_clustering_2016050501"/>
+
+For each node, specify the members to form the cluster with:
```

Comment:
For each machine? (we use both node, machine in this doc. we should use either node or machine, not both)